node/impl/storminer.go

```diff
@@ -49,54 +48,63 @@ import (
 type StorageMinerAPI struct {
 	common.CommonAPI
 
-	SectorBlocks *sectorblocks.SectorBlocks
-
-	PieceStore        dtypes.ProviderPieceStore
-	StorageProvider   storagemarket.StorageProvider
-	RetrievalProvider retrievalmarket.RetrievalProvider
-	Miner             *storage.Miner
-	BlockMiner        *miner.Miner
-	Full              api.FullNode
-	StorageMgr        *sectorstorage.Manager `optional:"true"`
-	IStorageMgr       sectorstorage.SectorManager
-	*stores.Index
-	storiface.WorkerReturn
-	DataTransfer  dtypes.ProviderDataTransfer
-	Host          host.Host
-	AddrSel       *storage.AddressSelector
-	DealPublisher *storageadapter.DealPublisher
-
-	Epp gen.WinningPoStProver
+	Full        api.FullNode
+	LocalStore  *stores.Local
+	RemoteStore *stores.Remote
+
+	// Markets
```

Comment:
We should split submodule APIs like https://github.com/filecoin-project/lotus/pull/6627, but I'm ok with that being done after this PR is merged